bridges/snowbridge/primitives/core/src/location.rs

```diff
@@ -94,6 +95,12 @@ impl DescribeLocation for DescribeTokenTerminal {
 	}
 }
 
+
+// Convert ERC20 token address to a location that can be understood by Assets Hub.
+pub fn convert_token_address(network: NetworkId, token: H160) -> Location {
+	Location::new(2, [GlobalConsensus(network), AccountKey20 { network: None, key: token.into() }])
+}
+
```

Comment:
I think this is not compatible anymore with native ETH: https://github.com/paritytech/polkadot-sdk/pull/6855/files#diff-5a39b1d9ef60342361183a0aed74e5dae96dad6a973162b59c593705a31e3314R398
```suggestion
```